samples/compose-browser/src/main/java/org/mozilla/samples/compose/browser/settings/SettingsScreen.kt

```diff
@@ -15,5 +20,20 @@ import androidx.compose.runtime.Composable
 fun SettingsScreen() {
     Column {
         Text("Settings")
+
+        components().icons.Loader(
+            url = "https://www.mozilla.org"
+        ) {
+            Placeholder {
+                Text("Loading icon...")
+            }
+            WithIcon { icon ->
+                Image(icon.painter, contentDescription = null)
+
+                // icon.color
+                // icon.maskable
+                // icon.source
```

Comment:
nit: Is this something that we want to address now ? :)

Reply:
Ah, Sorry, this was my test code and should not be in this commit. :)